onetimepass/enum.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import enum
+from typing import Any
+
+
+class StrEnum(str, enum.Enum):
+    pass
+
+
+class CaseInsensitiveStrEnum(StrEnum):
+    @classmethod
+    def _missing_(cls, value: Any) -> CaseInsensitiveStrEnum | None:
+        for member in cls:
+            if member.value == str(value).upper():
```

Comment:
this may not be a necessary change but we can `member.value` also be normalised

Reply:
done ✅